acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/BesuNode.java

```diff
@@ -60,7 +84,66 @@ public String getExternalJsonRpcUrl() {
     return "http://127.0.0.1:" + container.getMappedPort(JSON_RPC_PORT);
   }
 
+  public String getInternalEngineJsonRpcUrl() {
+    return "http://" + nodeAlias + ":" + ENGINE_JSON_RPC_PORT;
+  }
+
   public String getRichBenefactorKey() {
     return "0x8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63";
   }
+
+  public static class Config {
+    private final Map<String, Object> configMap = new HashMap<>();
+    private String genesisFilePath = "besu/depositContractGenesis.json";
+
+    public Config() {
+      configMap.put("rpc-http-enabled", true);
+      configMap.put("rpc-http-port", Integer.toString(JSON_RPC_PORT));
+      configMap.put("rpc-http-cors-origins", new String[] {"*"});
+      configMap.put("host-allowlist", new String[] {"*"});
+      configMap.put("miner-enabled", true);
+      configMap.put("miner-coinbase", "0xfe3b557e8fb62b89f4916b721be55ceb828dbd73");
+      configMap.put("genesis-file", "/genesis.json");
+    }
+
+    public BesuNode.Config withGenesisFile(final String genesisFilePath) {
+      this.genesisFilePath = genesisFilePath;
+      return this;
+    }
+
+    public BesuNode.Config withRpcHttpApi(final String[] rpcHttpApis) {
+      configMap.put("rpc-http-api", rpcHttpApis);
+      return this;
+    }
+
+    public BesuNode.Config withDefaultEngineRpcHttpPort() {
+      configMap.put("engine-rpc-http-port", Integer.toString(ENGINE_JSON_RPC_PORT));
+      return this;
+    }
+
+    public BesuNode.Config withEngineHostAllowList(final String[] engineHostAllowList) {
```

Comment:
nit: Using varargs here would avoid having to explicitly create an array when calling

```suggestion
    public BesuNode.Config withEngineHostAllowList(final String... engineHostAllowList) {
```

Reply:
Nice tip, thanks :) I've removed this method as this is now done in `withMergeSupport`